src/components/marketplace/buy_sell.tsx

```diff
@@ -217,25 +219,38 @@ class BuySell extends React.Component<Props, State> {
                             </FieldContainer>
                         </>
                     )}
-                    <Button theme="secondary" onClick={tab === Tab.Buy ? this.buy : this.sell}>
-                        {tab === Tab.Buy ? 'Buy' : 'Sell'}{' '}
+                    {orderType === OrderType.Limit ? <OrderDetails /> : null}
```

Comment:
Minor coment: probably the `<OrderDetails />` can be included at the end fragment above, below the closing `</FieldContainer>` in line 219.